Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+FROM elixir:1.10-alpine
```

Comment:
On the master branch Elixir 1.11 is now required. Since the official Docker image is not yet available, this could be used instead: ` FROM hexpm/elixir:1.11.0-erlang-23.1-alpine-3.12.0 AS builder `.